### DISCUSSION TITLE
EGLO Zigbee Remote 99099

### DISCUSSION THREAD

**Discussion by m1k3f15h** [arendst/Tasmota#18154] [Q&A]
Hy Guys

Did anyone use such a Remote with Tasmota Zigbee?
[https://www.eglo.com/at/fernbedienung-connect-z-99099.html](url)

Greetings from Austria
Mike

**Comment by grginterton** [comment]
Hi,
I'm facing the same challenge, did you figure out something?

Thanks a lot,
Gergő
